### DISCUSSION TITLE
Command Line to Start Slideshow

### DISCUSSION THREAD

**Discussion by Sk1Man** [torum/Image-viewer#22] [Ideas]
Would enjoy the ability to start the program with the InFrame Slideshow running automatically so that I can have it launch with windows and autostart... Same as pressing the "S" key with the program open. This would be nice with the Fullscreen start command as well.

**Comment by torum** [comment]
Hi! 
That sounds great and simple enough. I'll try in a few days. Thank you for the feedback!

**Comment by torum** [comment]
I think it is done. [v1.3.8](https://github.com/torum/Image-viewer/releases/tag/v1.3.8)

**Reply by Sk1Man** [reply]
Dude! That's amazing! Thanks for updating it so quickly, it's exactly what I was hoping for! I got on the Microsoft Store this morning and updated the app and it's working perfectly. I created a shortcut in the Windows Startup folder with the switches "ImageViewer.exe -i 60 -a on -s on -y on 'C:\<path>'" and it launches on login and starts my photo slideshow automatically.

**Reply by torum** [reply]
I'm glad you liked!
